tests/bindings/javascript/analyserexternalvariable.test.js

```diff
@@ -13,3 +13,64 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+
+const loadLibCellML = require('libcellml.js/libcellml.common')
+const { hhSquidAxon1952 } = require('./resources')
+
+let libcellml = null
+
+describe("Analyser External Variable tests", () => {
+    let m
+
+    beforeAll(async () => {
+        libcellml = await loadLibCellML()
+    });
+    beforeEach(() => {
+        const p = new libcellml.Parser()
+        m = p.parseModel(hhSquidAxon1952)
+    });
+    test("Checking Analyser.External Variable variable.", () => {
+        aev = new libcellml.AnalyserExternalVariable(m.componentByName("membrane", true).variableByName("V"))
+
+        expect(aev.variable().name()).toBe("V")
+    })
+    test("Checking Analyser.External Variable dependency.", () => {
+        aev = new libcellml.AnalyserExternalVariable(m.componentByName("membrane", true).variableByName("V"))
+
+        expect(aev.dependencyCount()).toBe(0)
+
+        const dependentVariable1 = m.componentByName("membrane", true).variableByName("i_Na")
+        const dependentVariable2 = m.componentByName("membrane", true).variableByName("i_K")
+        const dependentVariable3 = m.componentByName("membrane", true).variableByName("i_L")
+
+        expect(aev.addDependency(dependentVariable1)).toEqual(true)
+        expect(aev.containsDependencyByModel(m, "membrane", "i_Na")).toEqual(true)
```

Comment:
I find `containsDependencyByModel()` to be a strange name for that function. What about `containsDependencyByComponentAndVariableStrings()`, or something like that?

Reply:
The name of the method is given by the first differing argument of the overloaded method.  This approach has been used for all the overloaded API methods.

Reply:
When I see `containsDependencyByModel()`, I understand that I need to pass one argument: a model. Yet, I actually need to pass three of them: a model (as an object), as well as both a component and a variable (both as strings). So, tha API is confusing to me.

(My original suggestion is also wrong since I didn't account for the model parameter. So, if anything, I would probably have something like `containsDependencyByModelComponentAndVariable()`.)